app/assets/javascripts/application.js

```diff
@@ -3,6 +3,99 @@
 //= require turbolinks
 //= require_tree .
 
+// videocall
+
+let remoteContainer;
+let canvasContainer;
+
+document.addEventListener('turbolinks:load', event => {
```

Comment:
We are using the same function for Stripe down below (line 128), please use one function for both !

Reply:
Good point, fixed!